src/lake/Lake/CLI/Init.lean

```diff
@@ -300,6 +327,7 @@ def init (name : String) (tmp : InitTemplate) (lang : ConfigLang) (env : Lake.En
   validatePkgName name
   IO.FS.createDirAll cwd
   initPkg cwd (stringToLegalOrSimpleName name) tmp lang env
+  createLeanActionWorkflow cwd
```

Comment:
This should be part of `initPkg` rather than duplicated in `init` and `new`.

Reply:
Ah, okay. Done :+1: